### PR TITLE
feat(server): improved output messages for model operations

### PR DESCRIPTION
**Pull request by ahmedtadde**
## Feature or Problem
Outputs of the `wash app` commands (mainly `put` and `deploy`) do not include the manifest name. Including the manifest name is an overall DX improvement of the `wash` cli.

## Related Issues
#175 

## Release Information
v0.6.0

## Consumer Impact
just a QOL change for the `wash` cli

## Testing
Built on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

### CODE REVIEW

**Comment by ahmedtadde** [comment]
@brooksmtownsend  I tried testing the changes locally by 
- cloning the `wash` cli locally and updating [the wadm dependency ](https://github.com/wasmCloud/wash/blob/9daf0db964b55c4182164d82f604ed31e3f420e0/Cargo.toml#L150) to point to the `wadm` [branch with the changes](https://github.com/ahmedtadde/wadm/tree/feat/echo-manifest-name-for-some-model-ops). 
- I built the `wash` cli with `cargo build --release` 
- ran `./target/release/wash up` and then `./target/release/wash deploy wadm.yaml` (for the default/template `hello` actor). The output of `deploy` remains unchanged to my surprise. 

if it is not much trouble, how should i go about testing the changes locally?

File: src/server/handlers.rs [hunk 1]
```diff
@@ -333,7 +337,7 @@ impl<P: Publisher> Handler<P> {
                 Ok(_) => {
                     DeleteModelResponse {
                         result: DeleteResult::Deleted,
-                        message: "All models deleted".to_string(),
+                        message: format!("Successfully deleted model {}", name),
```
Comment (brooksmtownsend) [inline src/server/handlers.rs:340]: nit: this can be done with `{name}` inside the format args

File: src/server/handlers.rs [hunk 2]
```diff
@@ -388,7 +395,10 @@ impl<P: Publisher> Handler<P> {
                             .await
                             .map(|_| DeleteModelResponse {
                                 result: DeleteResult::Deleted,
-                                message: "Last model version deleted".to_string(),
+                                message: format!(
+                                    "Successfully deleted last version of model {}",
+                                    name
```
Comment (brooksmtownsend) [inline src/server/handlers.rs:400]: nit: this can be done with `{name}` inside the format args

File: src/server/handlers.rs [hunk 3]
```diff
@@ -636,7 +650,7 @@ impl<P: Publisher> Handler<P> {
                 .await
                 .map(|_| DeployModelResponse {
                     result: DeployResult::Acknowledged,
-                    message: "Undeployed model".to_string(),
+                    message: format!("Successfully undeployed model {}", name),
```
Comment (brooksmtownsend) [inline src/server/handlers.rs:653]: nit: this can be done with `{name}` inside the format args

File: src/server/handlers.rs [hunk 4]
```diff
@@ -649,7 +663,7 @@ impl<P: Publisher> Handler<P> {
             trace!("Manifest was already undeployed");
             DeployModelResponse {
                 result: DeployResult::Acknowledged,
-                message: "Undeployed model".to_string(),
+                message: format!("Model {} was already undeployed", name),
```
Comment (brooksmtownsend) [inline src/server/handlers.rs:666]: nit: this can be done with `{name}` inside the format args

File: src/server/handlers.rs [hunk 5]
```diff
@@ -732,7 +746,7 @@ impl<P: Publisher> Handler<P> {
             // case we unwrap to nothing
             serde_json::to_vec(&StatusResponse {
                 result: StatusResult::Ok,
-                message: "Successfully fetched status".to_string(),
+                message: format!("Successfully fetched status for model {}", name),
```
Comment (brooksmtownsend) [inline src/server/handlers.rs:749]: nit: this can be done with `{name}` inside the format args

**Review by brooksmtownsend — APPROVED** [review]
This looks great! I added some nits that are totally optional, going to get through re-running the e2e flaky tests first and they can be addressed in the future since there's no functional difference 😄

**Comment by ahmedtadde** [comment]
oh, I am just now seeing the comments. I would have made those changes but thanks for the merge.

**Comment by brooksmtownsend** [comment]
Not at all, I left them just a few minutes before merging 😄 Feel free to do a followup PR but those nits are really really minor

**Pull request merged**